csatar-plugins/csatar/components/partials/formButton.htm

```diff
@@ -21,6 +22,9 @@
             data-placement=top
             title="{{ tooltip }}"
         {% endif %}
+        {% if dataRequest %}
+            data-request="{{ dataRequest }}"
+        {% endif %}
```

Comment:
Van egy dataRequestButton partial is, felparamétereze csomó data-request-el kapcsolatos opcióval, biztos kell ennek a formButton-nak a data-request paraméter? Nem lehetne a másikat használni?